test/plots/bigint.ts

```diff
@@ -25,3 +26,19 @@ export async function bigintOrdinal() {
 export async function bigintStack() {
   return Plot.barY(integers, {x: (d, i) => i % 5, y: "big1"}).plot();
 }
+
+export async function bigintNormalize() {
+  const table = await Arrow.tableFromIPC(fetch("data/height_frequency.arrow"));
+  return Plot.plot({
+    height: 500,
+    x: {percent: true, grid: true},
+    y: {domain: [1.3, 2.21]},
```

Comment:
If you remove the y.domain here, the chart works in preview but crashes in node tests. I don't understand what the difference might be.

Reply:
![Capture d’écran 2024-09-09 à 19 39 39](https://github.com/user-attachments/assets/88e62089-e32c-4366-b3a3-6a0d85cf55bd)

This is how the chart looks like in preview. But it crashes in node. Is apache-arrow behaving differently in the browser?

Reply:
OK the test is now fixed — I just don't use a file and generate the arrow structure on the fly.